client/html/src/Client/Html/Catalog/Detail/Standard.php

```diff
@@ -183,6 +183,8 @@ public function getBody( string $uid = '' ) : string
 			{
 				$error = array( $context->getI18n()->dt( 'controller/frontend', $e->getMessage() ) );
 				$view->detailErrorList = array_merge( $view->get( 'detailErrorList', [] ), $error );
+
+                if ( $e->getCode() === 404 ) { throw $e; }
```

Comment:
Can you please fix indentions (one tab per indention)?

Reply:
Fixed